scripts/remove_after_use/fix_draft_node_permissions.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import sys
+
+from website.app import setup_django
+setup_django()
+
+from django.db import transaction
+from osf.models import EmbargoTerminationApproval
+from osf.models import Sanction
+import datetime
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+from scripts import utils as scripts_utils
+
+from osf.models import DraftRegistration
+
+
+def main(dry=True):
+    """
+    for 0199_draft_node_permissions we had a bug where permissions groups were made, but not actually populated leading
+    to a widespread permissions errors for draft registrations made prior to the migration. This should fix that.
+    Migration ran at 2120 (EDT).
+    """
+    date_of_migration = datetime.datetime(2020, 3, 24, 21, 20)
+
+    bugged_regs = DraftRegistration.objects.filter(created__lte=date_of_migration)
+
+    with transaction.atomic():
+        for reg in bugged_regs:
+            draft_perm_groups = reg.group_objects.order_by('name')
+            node_perm_groups = reg.branched_from.group_objects.order_by('name')
+
```

Comment:
Does this also need to update `draft.contributors`?

Reply:
Nope contributors is just a union of the groups.

Reply:
https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/registrations.py#L564-L566